crates/dojo-core/src/test_utils.cairo

```diff
@@ -68,34 +68,34 @@ fn spawn_test_world(models: Array<felt252>) -> IWorldDispatcher {
 }
 
 
-const GAS_OFFSET: felt252 = 0x1_000000_000000_000000_000000_000000; // 15 bajtów
+#[derive(Drop)]
+struct GasCounter {
+    start: u128,
+}
 
-/// Measures gas used after previous measurement and prints it
-///
-/// # Arguments
-///
-/// * `start` - gas before measurement
-/// * `name` - name of test, at most 15 bytes, will be padded with spaces
-fn end(start: u128, name: felt252) {
-    let gas_after = testing::get_available_gas();
-    gas::withdraw_gas().unwrap();
-    let mut name: u256 = name.into();
+#[generate_trait]
+impl GasCounterImpl of GasCounterTrait {
+    fn start() -> GasCounter {
+        let start = testing::get_available_gas();
+        gas::withdraw_gas().unwrap();
+        GasCounter { start }
+    }
 
-    // overwriting zeros with spaces
-    let mut char = 0;
-    loop {
-        if char == 15 {
-            break;
-        }
-        // if given byte is zero
-        if shl(0xff, 8 * char) & name == 0 {
-            name = name | shl(0x20, 8 * char); // set space
-        }
-        char += 1;
-    };
+    fn end(self: GasCounter, name: ByteArray) { 
+        let end = testing::get_available_gas();
+        let gas_used = self.start - end;
+
+        println!("# GAS # {}: {}", GasCounterImpl::pad_start(name, 18), gas_used);
+        gas::withdraw_gas().unwrap();
+    }
 
-    let name: felt252 = (name % GAS_OFFSET.into()).try_into().unwrap();
-    // Q?: What's this 1070 value that needed to be adjusted?
```

Comment:
I'm still not sure of why we were removing this `1070` before. May be not relevant anymore and was only for display?